src/main/java/frc/robot/constants/DriveConstants.java

```diff
@@ -39,7 +39,7 @@ public static final class Module {
     public static final int TALON_TURN_MOTOR_3 = 2;
     public static final int TALON_CANCODER_3 = 23;
 
-    public static final double WHEEL_RADIUS = Units.inchesToMeters(1.906);
+    public static final double WHEEL_RADIUS = Units.inchesToMeters(1.9375);
```

Comment:
We were seeing alignment issues all comp. I specifically remember an offset in Y during auto.

While I don't think this is the ultimate root cause, it could be a factor. It's not required to revert for this review, I would just keep an eye on it.